packages/table/src/interactions/reorderable.tsx

```diff
@@ -130,6 +137,13 @@ export class DragReorderable extends React.Component<IDragReorderable, {}> {
         } else {
             // select the new region to avoid complex and unintuitive UX w/r/t the existing selection
             this.props.onSelection([region]);
+            const focusCellCoordinates = Regions.getFocusCellCoordinatesFromRegion(region);
+            const fullFocusCellCoordinates: IFocusedCellCoordinates = {
+                col: focusCellCoordinates.col,
+                focusSelectionIndex: 1,
+                row: focusCellCoordinates.row,
+            };
+            this.props.onFocus(fullFocusCellCoordinates);
```

Comment:
i would write this entire block as simply

```js
// argument to function will be typed as IFocusedCellCoordinates, so you don't have to
this.props.onFocus({ ...focusedCellCoordinates, focusSelectionIndex: 1 })
```

Reply:
Apparently Chris has an incoming helper instead, see above -- going to use that. Good point though, your way here is more succinct.

Reply:
his helper does precisely what i wrote above. i honestly don't see a need to wait for the helper when the spread is so simple.

Reply:
Yeah @gscshoyru feel free to just do this one-liner for now

Reply:
Also, @gscshoyru did we mean to change `focusSelectionIndex` to `1` instead of `0`? There's only one region in the new `selectedRegions` array, so it seems like `0` is correct. LMK if that's wrong.